arc/recommendations/generic_analyzer.py

```
@@ -0,0 +1,437 @@
+""")
+        return "\n".join(formatted)
+    
+    def _parse_llm_recommendations(self, response: str) -> List[GenericRecommendation]:
+        """Parse LLM response into structured recommendations."""
+        try:
+            # Extract JSON from response
+            import re
+            json_match = re.search(r'\[.*\]', response, re.DOTALL)
+            if json_match:
+                recommendations_data = json.loads(json_match.group())
+                
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)